addons/certificate/models/certificate.py

```diff
@@ -297,6 +334,18 @@ def _get_public_key_numbers_bytes(self, formatting='encodebytes'):
         )
 
     def _get_public_key_bytes(self, encoding='der', formatting='encodebytes'):
+        ''' Get the certificate public key bytes
```

Comment:
```suggestion
        ''' Get the certificate's public key bytes
```

Above you wrote the other way "Get X of Y". Here it's "Get Y's X". If you want it all more unified, you can maybe decide one over the other and apply it everywhere ? (just my personal opinion: "Get X of Y" looks better. So `Get the public key bytes of the certificate.`)